src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -120,13 +120,13 @@ export const NewProposal = ({ navigation, data, query }) => {
       id: data?.id,
       attributes: {
         translationsAttributes: {
-          title: newProposalData.title,
-          summary: newProposalData.summary,
-          description: newProposalData.description
+          title: newProposalData?.title,
+          summary: newProposalData?.summary,
+          description: newProposalData?.description
         },
-        tagList: newProposalData.tagList,
+        tagList: newProposalData?.tagList,
         termsOfService: termsOfService,
```

Comment:
`termsOfService: termsOfService` -> `termsOfService` ;)

Reply:
and this is also a boolean, i just see: `const [termsOfService, setTermsOfService] = useState(data?.termsOfService ?? false);`

so please change it to `hasAcceptedTermsOfService` or something like that more understandable

Reply:
7a3a9dacaabe27e7b1291911444f7d2b05906983